src/terminal.rs

```diff
@@ -1,25 +1,20 @@
-
 use std;
 use termion;
-
 use termrect::{RawPaintable, HasSize};
 use style::Style;
 use styledtext::StyledText;
 use std::io::Write;
 
 pub struct Terminal<W: Write> {
     size: (u32, u32),
-    current_style: Style,
```

Comment:
I know it's technically unused, but I was leaving this there as a todo. Style diff and minimal updates is something I want to do soon.

Reply:
Done!